terraform/modules/notification_queue/main.tf

```diff
@@ -1,22 +1,22 @@
 
-module "transfer_shoots_topic" {
+module "notification_topic" {
   source = "github.com/wellcomecollection/terraform-aws-sns-topic.git?ref=v1.0.1"
-  name = "transfer-shoots-${var.environment}"
+  name = "${var.action_name}-${var.environment}"
 }
 
 module "dlq_alarm_topic" {
   source = "github.com/wellcomecollection/terraform-aws-sns-topic.git?ref=v1.0.1"
-  name = "transfer-shoots-alarm-${var.environment}"
+  name = "${var.action_name}-alarm-${var.environment}"
 }
```

Comment:
Nitpick: do we need DLQ alarms? We'll probably be keeping an eye on the transfer/touch DLQs when we start a run